src/components/Links/GitHub/GitHub.css

```diff
@@ -0,0 +1,15 @@
+.github__link {
+  display: flex;
+  align-items: center;
+  gap: 0.5em;
+  padding: 0.125em;
+
+  span {
```

Comment:
Non-blocker, but what do you think about opening an issue to use [browserlist](https://github.com/browserslist/browserslist), so we can explicitly tell that we are targeting modern browsers for CSS Nesting ([not a super good coverage at](https://caniuse.com/css-nesting) the moment).

Reply:
Ok, that's a good idea!